pkg/scheduler/algorithm/generic.go

```diff
@@ -426,7 +426,7 @@ func (g *genericScheduler) subgroupClusters(sub *appsapi.Subscription, clusterSc
 		}
 
 		if subscriber.SubGroupStrategy == nil {
-			return nil, fmt.Errorf("groupStrategy filed can not be Empty")
+			return nil, fmt.Errorf("subGroupStrategy filed can not be Empty")
```

Comment:
Does this mean that all subscribers should set `SubGroupStrategy`?

Shall we add default values for `subscriber.SubGroupStrategy` when `spec.SchedulingBySubGroup` is set?

Reply:
And for such errors, we'd better send events as well.